controllers/home.js

```diff
@@ -5,7 +5,7 @@ const CLASSES = require('../classes.js');
 exports.get_home = function(req, res, next) {
     api_connection.getUser(req.session.user).then( 
         function(data) {
-            api_connection.getUserPlaylists(req.session.user).then(
+            api_connection.getUserPlaylists(req.session.user, {limit: 50}).then(
```

Comment:
API breaks if the user has more than 50 songs. The default was 20 before, this change, hopefully less of these issues with this higher limit.